monitoring/snippets/v3/cloud-client/quickstart_test.py

```diff
@@ -22,11 +22,9 @@
 PROJECT = os.environ["GOOGLE_CLOUD_PROJECT"]
 
 
-def test_quickstart(capsys):
+def test_quickstart(capsys) -> None:
```

Comment:
We probably don't need `capsys` anymore.

Reply:
done